www/index.html

```diff
@@ -0,0 +1,94 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Wildcard certs for your localhosts</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+        header {
+            background: rgb(11, 17, 43);
+            margin-bottom: 50px;
+            padding: 30px;
+            padding-right: 30px;
+            padding-left: 30px;
+            color: rgb(255, 255, 255);
+        }
+        body {
+            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+            font-size: 14px;
+            line-height: 1.42857;
+            color: rgb(51, 51, 51);
+            background-color: rgb(255, 255, 255);
+            margin: 0;
+        }
+        .container {
+            width: 80%;
+            max-width: 800px;
+            margin-right: auto;
+            margin-left: auto;
+            padding-left: 15px;
+            padding-right: 15px;
+        }
+        .markdown-body a {
+            color: rgb(3, 102, 214);
+            text-decoration: none;
+        }
+        header h1 a {
+            color: rgb(255, 255, 255);
+            text-decoration: none;
+        }
+        code {
+            background-color:#ddd;
+        }
+    </style>
+</head>
+<body>
+<header class="container-fluid">
+    <div class="container">
+        <h1 class="set_url">Domain here</h1>
+        <p>DNS service for local IP addresses with TLS/HTTPS support</p>
+    </div>
+</header>
+<main class="markdown-body container">
+
+    <h1 ><span class="set_url">Domain here</span> - DNS service for local IP addresses with TLS/HTTPS
+        support</h1>
+
+
+    <h2 id="what-is-this-about">What is this about</h2>
+    <p>In one of our projects we faced a problem, that we wanted to connect to websocket on a local network from
+        webpage, that was served from the Internet. In order to use secure web socket connection (wss://) we needed the
+        SSL certificate, but you can't get one for local IP's. The bottomline was that we could not change client
+        machines configuration (no /etc/hosts, no importing of own CA's).</p>
+
+    <p>So we came up with the idea, that we could use wildcard SSL certificate and are using
```

Comment:
I think it'd be good to add a disclaimer here that this is completely unsafe since the private keys are available, so people are aware that it's meant to be a useful solution for some cases but not a secure one.

Reply:
agreed!  fixed in 48bf0ca